components/contact.module.css

```diff
@@ -0,0 +1,108 @@
+.contact_section_container {
+	width: 100%;
+	height: 100%;
+	display: grid;
+	grid-template-columns: 100%;
+	grid-template-rows: 15% 10% 75%;
+	justify-content: center;
+	align-items: center;
+	padding: 0 3rem;
+}
+
+.thank_you {
+	text-align: center;
+	margin: 0;
+}
+
+.form {
+	display: flex;
+	/* position: relative; */
```

Comment:
Is this comment?? It can be deleted at last?

Reply:
We can delete this now or later.